dev-requirements.txt

```diff
@@ -1,27 +1,135 @@
 -i https://pypi.python.org/simple
+aiohttp==3.8.1; python_version >= '3.6'
```

Comment:
how come the dev requirements have changed so much?

Reply:
Likely because we now specifically depend on the dev version of `nucypher`. 

Admittedly I'm still playing around with this.  Currently, the tricky part is that for testing, since there is no federated mode, we need to use testerchain from the `nucypher` fixtures. Subsequently, nucypher contracts need to be compiled and deployed to the testerchain, and that requires two things:
1. py-solc-x: only installed for the `[dev]` installation version of `nucypher`
2. contract solidity files, only available for the editable `nucypher` dependency.

# 1 is achieved, but achieving # 2 above is causing some issues since editable support for a dependency is a minefield, and this is why the tests are failing.

If you have any insight there as well, please share 🙏 .